_posts/2019-01-23-multifile-python-functions.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Multifile python functions
+description: "Tidy python in an OpenFaaS world; or, what happens when you want to split your Python function into multiple modules?"
+date: 2019-01-23
+image: /images/multifile-python-function/pexels-book-stack-books-bookshop-264635.jpg
```

Comment:
Did you resize and down-size the binary as per the editorial guidelines?

Reply:
I missed that in the guide, thanks for catching it